uber/site_sections/accounts.py

```diff
@@ -20,8 +20,7 @@ def index(self, session, message=''):
                                .order_by(Attendee.first_name, Attendee.last_name).all(),
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), c.BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
-                for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                                              .filter(Attendee.email != '').all()
+                for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num).all()
```

Comment:
Yeah, having a bajillion unclaimed badges is my main concern for this. @Lusarien can you add that check to the query?

Further UX improvements can be added as issues so we can address them later - that's more of a front-end concern anyway.